metaflow/plugins/aip/aip.py

```diff
@@ -1062,8 +1062,9 @@ def _get_flow_labels(
         # the Zodiac service and team labels are added to the AIP pods and set. These labels are not added
         # by the AIP webhook to support user-supplied Zodiac service per AIP Notebook. Workflows launched
         # in project CICD profiles will still have these labels added via the AIP webhook.
-        if ZILLOW_ZODIAC_SERVICE and ZILLOW_ZODIAC_TEAM:
+        if ZILLOW_ZODIAC_SERVICE:
             ret_flow_labels[f"{zodiac_prefix}/service"] = ZILLOW_ZODIAC_SERVICE
+        if ZILLOW_ZODIAC_TEAM:
             ret_flow_labels[f"{zodiac_prefix}/team"] = ZILLOW_ZODIAC_TEAM
```

Comment:
fyi: these are applied at compile time during CICD.

Should these be injected by the webhook?

Reply:
I do want to link a workflow to a zodiac service at compile time. This ensures that the workflow manifest is linked to a service no matter where it is. 

I'm considering removing zodiac team label. The workflow - zodiac service association is much more stable then the workflow - zodiac team association, and zodiac team can be inferred from the zodiac service.